api/src/models/resource.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require('mongoose');
+
+const PhoneNumber = new mongoose.Schema({
+  phoneType: { type: String, required: true },
+  phoneNumber: { type: String, required: true },
+});
+
+const OtherContact = new mongoose.Schema({
+  roleOfContact: { type: String, required: true },
+  name: { type: String, required: true },
+  email: { type: String, required: false },
+  phoneNumber: { type: String, required: false },
+  note: { type: String, required: false },
+});
+
+const FinancialAid = new mongoose.Schema({
+  education: { type: String, required: false },
+  immigrationStatus: { type: String, required: false },
+  deadline: { type: String, required: false },
+});
+
+const Resource = new mongoose.Schema({
+  category: { type: String, required: true },
+  name: { type: String, required: true },
+  description: { type: String, required: true },
+  website: { type: String, required: false },
+  email: { type: String, required: false },
+  phoneNumbers: { type: [PhoneNumber], required: false },
+  otherContacts: { type: [OtherContact], required: false },
```

Comment:
Since we don't have another contact field, what if we change `otherContacts` to `contacts`, and then the schema `OtherContact` to `Contact`